dist/rx-player.js

```diff
@@ -16460,6 +16459,10 @@ var Adaptation = /*#__PURE__*/function () {
       this.isSignInterpreted = parsedAdaptation.isSignInterpreted;
     }
 
+    if (parsedAdaptation.label !== undefined) {
```

Comment:
We usually don't stage `dist` until the release. Could you unstage it?

Reply:
Sure! 

I wasn't really sure on what to do with the `dist`-directory. I'll revert the changes from this PR.